.github/CODEOWNERS

```diff
@@ -0,0 +1,61 @@
+# ------
+# DOCUMENTATION / EXAMPLES
+# https://docs.github.com/en/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/about-code-owners
+# ------
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# @global-owner1 and @global-owner2 will be requested for
+# review when someone opens a pull request.
+# * @global-owner1 and @global-owner2
+
+# Order is important; the last matching pattern takes the most
+# precedence. When someone opens a pull request that only
+# modifies JS files, only @js-owner and not the global
+# owner(s) will be requested for a review.
+# *.js    @js-owner
+
+# You can also use email addresses if you prefer. They'll be
+# used to look up users just like we do for commit author
+# emails.
+# *.go docs@example.com
+
+# In this example, @doctocat owns any files in the build/logs
+# directory at the root of the repository and any of its
+# subdirectories.
+# /build/logs/ @doctocat
+
+# The `docs/*` pattern will match files like
+# `docs/getting-started.md` but not further nested files like
+# `docs/build-app/troubleshooting.md`.
+# docs/*  docs@example.com
+
+# In this example, @octocat owns any file in an apps directory
+# anywhere in your repository.
+# apps/ @octocat
+
+# In this example, @doctocat owns any file in the `/docs`
+# directory in the root of your repository and any of its
+# subdirectories.
+# /docs/ @doctocat
+
+# -------
+# ACTUAL RULES
+# -------
+
+# -- NOTE ---
+# The people you choose as code owners must have write permissions for the repository. When the code owner is a team,
+# that team must have write permissions, even if all the individual members of the team already have write permissions
+# directly, through organization membership, or through another team membership.
+
+# --- GENERAL ---
+* @lightningterror
```

Comment:
fyi @lightningterror incase you don't want to be tagged on every PR (unless a later rule takes precedence)